docs/reference/indices/put-component-template.asciidoc

```diff
@@ -91,23 +91,25 @@ Name of the component template to create.
 {es} includes the following built-in component templates:
 
 // tag::built-in-component-templates[]
-- `logs-mappings`
-- `logs-settings`
-- `metrics-mappings`
-- `metrics-settings`
-- `metrics-tsdb-settings`
-- `synthetics-mapping`
-- `synthetics-settings`
+- `logs@mappings`
+- `logs@settings`
+- `metrics@mappings`
+- `metrics@settings`
+- `metrics@tsdb-settings`
+- `synthetics@mapping`
+- `synthetics@settings`
 // end::built-in-component-templates[]
 
 {fleet-guide}/fleet-overview.html[{agent}] uses these templates to configure
-backing indices for its data streams. If you use {agent} and want to overwrite
-one of these templates, set the `version` for your replacement template higher
-than the current version.
```

Comment:
@joshdover @dakrone it seems strange that we've ever recommended overriding the managed component templates. But maybe that comes from a time before Fleet added the `@custom` component templates.

Please have a look at what I'm proposing instead and let me know what you think.

Reply:
This history predates my time on Fleet, so it could be we didn't have that feature yet when this was written, but I believe it was in the first GA release of Fleet+Agent (7.14).

New copy looks good to me.

Reply:
Yes it's definitely from before we had the `@custom` thing, so I think it's okay now.